src/System.Collections/src/System.Collections.csproj

```diff
@@ -40,7 +42,7 @@
       <TargetGroup>uap101aot</TargetGroup>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap101aot'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uap101aot'">
```

Comment:
at least for now, uap101aot is the latest (it is the one pointed by uapaot), but yeah, in theory we want to go and change all of these conditions to the versionless TFMs assuming they don't ship as individual packages.